packages/client/src/generation/utils/buildDirname.ts

```diff
@@ -34,7 +34,16 @@ const fs = require('fs')
 
 config.dirname = __dirname
 if (!fs.existsSync(path.join(__dirname, 'schema.prisma'))) {
-  config.dirname = path.join(process.cwd(), ${JSON.stringify(pathToPosix(relativeOutdir))})
+  const alternativePaths = [
+    ${JSON.stringify(pathToPosix(relativeOutdir))},
+    ${JSON.stringify(pathToPosix(relativeOutdir).split('/').slice(1).join('/'))},
```

Comment:
This is the new path?
What does that "translate" to? Maybe a comment to describe for all the alternative paths as an example?

Reply:
Hey @millsp, wonder if you've seen this comment? (Tagging you in Slack as well)

Reply:
It will be equivalent to what we had before (see last picture [here](https://github.com/prisma/prisma/issues/19819#issuecomment-1597138099)).

Reply:
Maybe add a comment in the code?

Reply:
Done